SU-SWS/stanford_components
# EARTH-18 Link banner tweeks

# READY FOR REVIEW

# Summary
- Fixed rendering with fields
- Used `image` instead of `imgsrc` for region key

# Needed By (Date)
- End of sprint

# Urgency
- Some kinda

# Steps to Test

1. Checkout `EARTH-18-link-banner` branch of [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/44)
2. Checkout this branch.
3. `drush en stanford_link_banner -y`
4. Ensure patterns/link_banner page still looks correct
5. Create a complex page with a "Link Banner" paragraph item.
6. Ensure it looks about correct.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/44

# See Also
- [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/44)
